src/CoreWCF.Primitives/src/CoreWCF/Channels/BufferManager.cs

```diff
@@ -2,6 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System;
+using System.Buffers;
 using CoreWCF.Runtime;
```

Comment:
You've added this using but no code change

Reply:
done thanks